packages/trpc/server/routers/viewer/organizations/checkIfOrgNeedsUpgrade.handler.ts

```diff
@@ -12,7 +11,7 @@ type GetUpgradeableOptions = {
 };
 
 export async function checkIfOrgNeedsUpgradeHandler({ ctx }: GetUpgradeableOptions) {
-  if (!IS_TEAM_BILLING_ENABLED) return [];
+  // if (!IS_TEAM_BILLING_ENABLED) return [];
```

Comment:
do we need to keep this ?

Reply:
I'll undo this. I was just testing this banner